### DISCUSSION TITLE
What's the short version of --implicit-csv-header?

### DISCUSSION THREAD

**Discussion by aborruso** [johnkerl/miller#1046] [Q&A] [answered]
Hi,
in the [CSV flags](https://miller.readthedocs.io/en/latest/reference-main-flag-list/#csvtsv-only-flags) we have `--implicit-csv-header`  and `--headerless-csv-input`.

And we have `--hi` that is the short version of `--headerless-csv-input`.

What's the one for  `--implicit-csv-header`?

Thank you

**Comment by johnkerl** [comment, marked as answer]
@aborruso it's `--hi` -- these are all aliases for the same thing

https://github.com/johnkerl/miller/blob/main/internal/pkg/cli/option_parse.go#L2114-L2121
